examples/tfengine/app.hcl

```diff
@@ -20,46 +20,267 @@ template "app" {
   data = {
     constants = {
       shared = {
-        env_code        = "s"
-        folder_id       = "123"
-        billing_account = "000"
-        project_prefix  = "example-prefix"
-        state_bucket    = "example-state"
-        region          = "us-central1"
+        env_code            = "s"
+        folder_id           = "123"
+        billing_account     = "000"
+        project_prefix      = "example-prefix"
+        state_bucket        = "example-state"
+        bigquery_location   = "us-east1" # No BigQuery region in us-central1.
+        cloud_sql_region    = "us-central1"
+        cloud_sql_zone      = "a"
+        compute_region      = "us-central1"
+        gke_region          = "us-central1"
+        storage_location    = "us-central1"
```

Comment:
maybe at somepoint we should standardize the usage of both location and region. Maybe just use `region` and `zone` across all of these variables.

Reply:
The reason they are location in some cases is because it can be both regional or multiregional (in the case of storage) whereas for some it can only be region

Reply:
Got it. Maybe we can leave a comment for that.